plugin-server/package.json

```diff
@@ -8,8 +8,8 @@
         "test": "jest --runInBand --forceExit tests/**/*.test.ts",
         "test:postgres:1": "jest --runInBand --forceExit tests/ --testPathIgnorePatterns tests/clickhouse/ --testPathIgnorePatterns tests/postgres/ --testPathIgnorePatterns tests/shared/process-event.test.ts --testPathIgnorePatterns tests/worker/ingestion/event-pipeline",
```

Comment:
Shouldn't this be updated too?

Reply:
To what? We'll be removing these two lines after yeet project.

Reply:
Well, just that this one now seems pretty random, because the other three scripts already should be covering 100% of the suites. But yeah, we need to completely rename and reorder this anyway this week, so fine with running more than needed at the moment.